src/store/games/gamesReducer.js

```diff
@@ -0,0 +1,17 @@
+
+const INITIAL_STATE = {
```

Comment:
Do you still need this file if you're creating the reducer in gamesSlice.js?

Reply:
file removed